Dockerfile

```diff
@@ -18,8 +18,8 @@ SHELL ["/bin/bash", "--login", "-o", "pipefail", "-c"]
 # Install common OS packages
 RUN yum -y install \
         centos-release-scl \
-        epel-release \
-        https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-7-x86_64/pgdg-centos10-10-2.noarch.rpm && \
+        epel-release && \
+    rpm -i https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm && \
```

Comment:
Can't speak for the other changes, but :+1: on this. We had to do the same in ansible-consumer-finance.